srcpkgs/python3-prometheus_client/template

```diff
@@ -1,14 +1,19 @@
 # Template file for 'python3-prometheus_client'
 pkgname=python3-prometheus_client
-version=0.7.1
-revision=4
+version=0.9.0
+revision=1
 wrksrc="prometheus_client-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3"
+checkdepends="python3-pytest"
 short_desc="Python3 client for the Prometheus monitoring system"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="Apache-2.0"
 homepage="https://github.com/prometheus/client_python"
 distfiles="${PYPI_SITE}/p/prometheus_client/prometheus_client-${version}.tar.gz"
-checksum=71cd24a2b3eb335cb800c7159f423df1bd4dcd5171b234be15e3f31ec9f622da
+checksum=9da7b32f02439d8c04f7777021c304ed51d9ec180604700c1ba72a4d44dceb03
+
+do_check() {
+	pytest tests
+}
```

Comment:
You should be able to drop this and just add `make_check_target="tests"` under the `build_style` line.